drivers/ipvlan/ipvlan_joinleave.go

```diff
@@ -121,6 +121,7 @@ func (d *driver) Join(nid, eid string, sboxKey string, jinfo driverapi.JoinInfo,
 
 // Leave method is invoked when a Sandbox detaches from an endpoint.
 func (d *driver) Leave(nid, eid string) error {
+	defer osl.InitOSContext()()
```

Comment:
Ok. Actually we do not do that in the existing drivers (ex. overlay). But it's true that adding it does not hurt.